.github/PULL_REQUEST_TEMPLATE/feature-pr.md

```diff
@@ -1,102 +1,35 @@
-
-Contains changes that require updates to existing **_code_** and/or **_Velaptor product_** documentation.
-- [ ] Yes
-  - [ ] I have updated the documentation accordingly.
-    - Choose the type of documentation.
-    - [ ] Code Documentation
-      - [ ] A ![documentation-code-label](https://user-images.githubusercontent.com/85414302/154672489-8079ed03-b8ff-41ff-9864-1e2ae55300cc.png) label has been added to the pull request.
-    - [ ] Product Documentation
-      - [ ] A ![documentation-product-label](https://user-images.githubusercontent.com/85414302/154672508-5ac50eb7-67f8-4cdf-92b9-fe5fcbd93b14.png) label has been added to the pull request.
-- [ ] No
-</details>
-
+- [ ] The **_head(source)_** branch for this pull request is a **_feature_** branch with the correct naming syntax in the following example.
+  - A **_head(source)_** branch is the branch being merged into a target branch.
+    <details closed><summary>Example</summary>
 
-<details open><summary>🧪Unit Testing🧪</summary>
+      ``` xml
+      Syntax: feature/<issue-num>-<description>
+      Example: feature/123-my-feature
+      ```
+    </details>
 
-My change requires unit tests.
-- [ ] Yes
-- [ ] No
-</details>
+- [ ] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.
 
+💡For more info on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).
 
-<details open><summary>🧪Manual Testing🧪</summary>
+- [ ] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
+  - A **_base(target)_** branch is the branch that is a source branch being merged into it.
 
-- [ ] Yes
-- [ ] No
-</details>
+- [ ] Pull request title matches the title of the associated issue.
 
+- [ ] Associated issue exists and is linked to this pull request.
+  - One issue per pull request.
 
-<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
+- [ ] The labels attached to this PR match the labels attached to the associated issue.
 
-<!-- Go over all of the following points, and put an `x` in all the boxes that apply. -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] Pull request title matches the example below with the correct version.
-  - **Example:** 🚀Release To Preview - v1.2.3-preview.4
-- [ ] Issues exist and are linked to this pull request.
 - [ ] My code follows the coding style of this project.
   - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
+
+- [ ] I have manually tested my code changes to the best of my ability.
+
 - [ ] All tests passed locally.
   - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
```

Comment:
Misspelling of vice
```suggestion
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
```